_pages/policies.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: page
+title: Policies
+subtitle: Policies
+permalink: /policies/
```

Comment:
do we want to add it to the nav bar? do we want to link to it from somewhere (I think the latter is fine)

Reply:
yeah i think we want to reserve nav bar space for the schedule and speakers. I'm not super sure where's a good idea to link this

Reply:
maybe on the home page:

```
## Policies

Learn about our photography, alcohol, and other policies [here](/policies)!
```